extensions/deepcast/src/utils.ts

```diff
@@ -1,34 +1,58 @@
 import { getSelectedText, Clipboard, Toast, showToast, getPreferenceValues } from "@raycast/api";
 import got from "got";
+import { StatusCodes, getReasonPhrase } from "http-status-codes";
 
 interface Preferences {
   key: string;
-  pro: boolean;
 }
 
 export function getPreferences() {
   return getPreferenceValues<Preferences>();
 }
 
+function isPro(key: string) {
+  return !key.endsWith(":fx");
+}
+
+function gotErrorToString(error: unknown) {
+  // response received
+  if (error instanceof got.HTTPError) {
+    const { statusCode } = error.response;
+    if (statusCode === StatusCodes.FORBIDDEN) return "Invalid DeepL API key";
+    if (statusCode === StatusCodes.TOO_MANY_REQUESTS) return "Too many requests to DeepL API";
+    if (statusCode === 456)
+      return "DeepL API quota exceeded. The translation limit of your account has been reached. Consider upgrading your subscription.";
+    if (statusCode.toString().startsWith("5")) return "DeepL API is temporary unavailable. Please try again later.";
+
+    return `DeepL API returned ${statusCode} ${getReasonPhrase(statusCode)}`;
+  }
+
+  // request failed
+  if (error instanceof got.RequestError) return `Failed to send a request to DeepL API: ${error.code} ${error.message}`;
```

Comment:
Could you please rephrase this message to `Something went wrong when sending a request to the DeepL API. If you’re having issues, open an issue on GitHub and include following text: ${error.code} ${error.message}`

Reply:
All seems resolved in the latest commit :)